Tests/Shared.Specs/Specialized/TaskCompletionSourceAssertionSpecs.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions.Extensions;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs
+{
+    public class TaskCompletionSourceAssertionSpecs
```

Comment:
Please use `Arrange`, `Act` and `Assert` comments

Reply:
Do you really need these comments?
I added blank lines for separation.
Is that not enough?

Reply:
It's more a convention in our code base.